carapace-server/src/main/java/org/carapaceproxy/server/RequestHandler.java

```diff
@@ -836,6 +837,9 @@ public String getProperty(String name) {
                     return request.method().name();
                 case PROPERTY_CONTENT_TYPE:
                     return request.headers().get(javax.ws.rs.core.HttpHeaders.CONTENT_TYPE, "");
+                case PROPERTY_SERVER_IP:
```

Comment:
maybe this should be PROPERTY_LISTENER_ADDRESS and the property below PROPERTY_LISTENER_HOST_PORT